cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Not sure whether we want/need any special config settings, but [here are the options](https://docs.cypress.io/guides/references/configuration.html#Options) just for reference

Reply:
Great suggestion, thanks for the link Nora!